### PR TITLE
Add a passing test suite for PATCH /api/articles/:article_id

### PR DESCRIPTION
**Pull request by stevenwu99**
I have completed Task 8 for PATCH /api/articles/:article_id.

Please check

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -254,3 +254,108 @@ describe ('POST /api/articles/:article_id/comments',() => {
        });
 })
 
+//Task 8 PATCH /api/articles/:article_id
+describe ('PATCH /api/articles/:article_id',() => {
+    test ('201:should return a update article with correct vote when passed a positive vote number', () => {
```
Comment (johnhutcheon) [inline __tests__/app.test.js]: These 201s should all be 200s 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -254,3 +254,108 @@ describe ('POST /api/articles/:article_id/comments',() => {
...
+    test ("400: should return an error if invalid article_id", () => {
+        const  newVote = {inc_votes:12};
+        return request(app)
+            .patch("/api/articles/nonsense")
+            .send(newVote)
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Bad request");
+            });
+    });
```
Comment (johnhutcheon) [inline __tests__/app.test.js:358]: Really great testing Steven! 🙌

**Review by johnhutcheon — APPROVED** [review]
Really good work Stephen! Just please amend the 201 tests to be 200 as per my comment, then happy for you to merge! 😄

File: __tests__/app.test.js (hunk 2 above)
Comment (stevenwu99) [reply __tests__/app.test.js:358]: Done

**Pull request merged**